### ISSUE TITLE
Unable to define Camera on FlameBlocGame

### ISSUE THREAD

**Issue by abgutierrez** [flame-engine/flame#1330]
# Current bug behaviour

The Constructor used for FlameBlocGame class does not include camera Parameter (as parent class does)
# Expected behaviour

Define cameraConfig on FlameBlocGame as FlameGame does .

# Steps to reproduce

- Create class BaseGame extending FlameBlocGame
- there's no way to define cameraConfig because we cannot send parameter to child

# Flutter doctor output

```
Output of: flutter doctor -v
```
[✓] Flutter (Channel stable, 2.5.3, on macOS 12.0.1 21A559 darwin-x64, locale
    en-UY)
    • Flutter version 2.5.3 at /Users/agustin/development/flutter-2.5.3
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 18116933e7 (3 months ago), 2021-10-15 10:46:35 -0700
    • Engine revision d3ea636dc5
    • Dart version 2.14.4

[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
    • Android SDK at /Users/agustin/Library/Android/sdk
    • Platform android-31, build-tools 31.0.0
    • Java binary at: /Applications/Android
      Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.8+10-b944.6916264)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 13.2.1, Build version 13C100
    • CocoaPods version 1.11.2

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.8+10-b944.6916264)

[✓] VS Code (version 1.63.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.32.0

[✓] Connected device (1 available)
    • Chrome (web) • chrome • web-javascript • Google Chrome 97.0.4692.99

• No issues found!

# More environment information

Platform affected:  web
Flame version: 1.0.0
# Log information

```
Enter log information in this code block
```

# More information

**Comment by st-pasha** [comment]
what's a `CameraConfig`?

**Comment by abgutierrez** [comment]
Sorry, I was confused with another camera class used on Bonfire. I was talking about Camera class used on FlameGame constructor to create CameraWrapper I also created a new  [PR 1331](https://github.com/flame-engine/flame/pull/1331) to add this feature.

**Comment by spydon** [comment]
Closed by  #1331
